tests/robot-tests/tests/libs/common.robot

```diff
@@ -93,6 +94,11 @@ user opens chrome headlessly
     Call Method    ${c_opts}    add_argument    ignore-certificate-errors
     Call Method    ${c_opts}    add_argument    log-level\=3
     Call Method    ${c_opts}    add_argument    disable-logging
+    Call Method    ${c_opts}    add_argument    download.default_directory=${DOWNLOADS_DIR}
+
+    ${prefs}=    Create Dictionary
```

Comment:
Looks like I get the following error due to this when running this test against dev (using 3.10 & the latest chromedriver): 

``` 
Calling method 'add_argument' failed: TypeError: ArgOptions.add_argument() got an unexpected keyword argument 'download.default_directory'
```